src/main/scala/codacy/plugins/test/ITest.scala

```diff
@@ -149,9 +149,13 @@ trait ITest {
       }
 
     if (fileErrorsResults.nonEmpty) {
-      Printer.red(s"Some files were not analysed because the tool failed analysing them!")
-      Printer.white(fileErrorsResults.map(fe => s"* File: ${fe.filename}, Error: ${fe.message}").mkString("\n"))
+      error(s"Some files were not analysed because the tool failed analysing them!")
+      info(fileErrorsResults.map(fe => s"* File: ${fe.filename}, Error: ${fe.message}").mkString("\n"))
     }
     issuesResults
   }
+
+  protected def toRelativePath(rootPath: String, absolutePath: String) = {
+    absolutePath.stripPrefix(rootPath).stripPrefix(JFile.separator)
```

Comment:
there is a `relativize` method coming from `java.nio.Path` without having to roll our own